Nova/src/main/kotlin/xyz/xenondevs/nova/integration/protection/plugin/IridiumSkyblock.kt

```diff
@@ -0,0 +1,57 @@
+package xyz.xenondevs.nova.integration.protection.plugin
+
+import com.iridium.iridiumskyblock.PermissionType
+import com.iridium.iridiumskyblock.database.Island
+import com.iridium.iridiumskyblock.database.User
+import me.ryanhamshire.GriefPrevention.GriefPrevention
+import org.bukkit.Bukkit
+import org.bukkit.Location
+import org.bukkit.OfflinePlayer
+import org.bukkit.entity.Entity
+import org.bukkit.inventory.ItemStack
+import xyz.xenondevs.nova.api.protection.ProtectionIntegration
+import xyz.xenondevs.nova.integration.InternalIntegration
+import java.util.*
+
+import com.iridium.iridiumskyblock.IridiumSkyblock as ISkyBlock
+
+
+internal object IridiumSkyblock : ProtectionIntegration, InternalIntegration {
+
+    private val IRIDIUMSKYBLOCK = if (Bukkit.getPluginManager().getPlugin("IridiumSkyblock") != null) GriefPrevention.instance else null
```

Comment:
Copy and paste error with ``GriefPrevention.instance`` and why do you even store the IridiumSkyblock instance if you never use it?

Reply:
Sorry for making a silly mistake, I didn't read the source code extensively so I just drew it up as I went along.